examples/bayesian_regression.py

```diff
@@ -20,7 +20,8 @@
 def build_linear_dataset(N, p, noise_std=0.01):
     X = np.random.rand(N, p)
     # use random integer weights from [0, 7]
-    w = np.random.randint(8, size=p)
+    w = np.random.randint(4, size=p)
+    print('w = {}'.format(w))
```

Comment:
Was this debug statement accidentally committed?

Reply:
since `w` is selected randomly, i thought it would be helpful to output the `w` selected, otherwise the user would have no way of comparing to the ground truth